src/gui/widgets/LcdSpinBox.cpp

```diff
@@ -121,15 +123,19 @@ void LcdSpinBox::mouseMoveEvent( QMouseEvent* event )
 {
 	if( m_mouseMoving )
 	{
-		int dy = event->globalY() - m_origMousePos.y();
-		if( event->modifiers() & Qt::ShiftModifier )
-			dy = qBound( -4, dy/4, 4 );
-		if( dy > 1 || dy < -1 )
+		int dy = event->globalY() - m_lastMousePos.y();
+		if( dy )
 		{
-			model()->setInitValue( model()->value() -
-						dy / 2 * model()->step<int>() );
+			float fdy = static_cast<float>(dy);
+			if( event->modifiers() & Qt::ShiftModifier )
+				fdy = qBound( -4.f, fdy/4.f, 4.f );
+			float floatValNotRounded = model()->value() + m_remainder -
+										fdy / 2.f * model()->step<int>();
```

Comment:
```suggestion
			float floatValNotRounded =
				model()->value() + m_remainder - fdy / 2.f * model()->step<int>();
```

Or something similar. All those extra tabs aren't needed IMO.

Reply:
Indeed, it looks better when you break at the equal sign :+1: